client/my-sites/domains/domain-management/controller.jsx

```diff
@@ -74,7 +73,6 @@ export default {
 				analyticsTitle="Domain Management > Edit"
 				component={ DomainManagement.Edit }
 				context={ pageContext }
-				needsCart
```

Comment:
From what I can tell, none of these actually use a `cart` prop apart from `DomainManagement.List`.